frontend/src/apiMocks/mockData.ts

```diff
@@ -187,6 +187,7 @@ export const domainaddresses: Record<
       num_forwarded: 0,
       num_spam: 0,
       type: "custom",
+      used_on: "",
```

Comment:
I get a `null` when this is unset for a fresh domain. Do we need to expect that on the React side, or enforce an empty string it on the Relay side?

Reply:
If it's supposed to be able to be `null` (i.e. the React side should expect that), we'll want to add ` | null` to the type in `frontend/src/hooks/api/aliases.ts`.

Reply:
I followed what I did on https://github.com/mozilla/fx-private-relay/pull/1713/files. Was that right? :)

Reply:
Updated type to have `| null` too.